RAKE/RAKE.py

```diff
@@ -27,19 +27,34 @@ def FoxStopList():
     return FoxStopList.words()
 
 
-def load_stop_words(stop_word_file):
+def MySQLStopList():
+    from .stoplists import MySQLStopList
+    return MySQLStopList.words()
+
+
+def NLTKStopList():
+    from .stoplists import NLTKStopList
+    return NLTKStopList.words()
+
+
+def load_stop_words(stop_word_file, divide, delimiter):
     """
     Utility function to load stop words from a file and return as a list of words
     @param stop_word_file Path and file name of a file containing stop words.
     @return list A list of stop words.
     """
     stop_words = []
-    for line in open(stop_word_file):
-        for word in line.split():  # in case more than one per line
-            stop_words.append(word)
+    if divide:
```

Comment:
Sorry, that was more of a pseudocode example. Concretely I was picturing something like this:

```python
def load_stop_words(stop_word_file, delimiter='[\W\n]+'):
    with open(stop_word_file) as f:
       stop_words = re.split(delimiter, f.read())
    return [word for word in stop_words if word not in ('', ' ')]  # filters empty string matches 
```

That would accept a regex delimiter, defaulting to splitting on whitespace and newlines, and return a list of non-empty non-space words.